main/src/test/java/org/mobilitydata/gtfsvalidator/validator/ShapeIncreasingDistanceValidatorTest.java

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright 2020 Google LLC, MobilityData IO
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.validator;
+
+import com.google.common.collect.ArrayListMultimap;
+import com.google.common.collect.ListMultimap;
+import org.junit.Before;
+import org.junit.Test;
+import org.mobilitydata.gtfsvalidator.notice.DecreasingShapeDistanceNotice;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.table.GtfsShape;
+import org.mobilitydata.gtfsvalidator.table.GtfsShapeTableContainer;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+public class ShapeIncreasingDistanceValidatorTest {
+    @Mock
+    final GtfsShapeTableContainer mockShapeTable = mock(GtfsShapeTableContainer.class);
+    @InjectMocks
+    final ShapeIncreasingDistanceValidator underTest = new ShapeIncreasingDistanceValidator();
+
+    @Before
+    public void openMocks() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @Test
```

Comment:
Need a test for when two sequential shape distances are the same (should generate notice)

Reply:
Oh, I think there is a bug in the current implementation. Spec says they should increase, but current rule allows them to be the same. Please double-check the spec and behavior to make sure I have this right.

Reply:
Indeed, the specification states that: 

> Values must increase along with shape_pt_sequence;


Plus, `v1.4.0` used to enforce this: https://github.com/MobilityData/gtfs-validator/blob/76970f6ab56c885ef3fad951a6bff2531ae18a3e/usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/ValidateShapeIncreasingDistance.java#L52

So I conclude that having strictly increasing values for `shape_dist_traveled` is the correct behavior. Will check the spec team thought.

Reply:
Per discussion with the spec team: 
Two shape points can have the same `shapes.shape_dist_traveled` value but in that occasion, a notice could be could be issued.

The type of notice (warning or error)  would depend on the distance between the two shape points:
- less than 10 meters: a warning
- more dans 10 meters: an error.

This distance threshold could be user configurable.